gcp/api/server.py

```diff
@@ -528,7 +553,7 @@ def to_response(b):
 def bug_to_response(bug, include_details=True):
   """Convert a Bug entity to a response object."""
   if include_details:
-    return bug.to_vulnerability(include_source=True)
+    return bug.to_vulnerability(include_source=True, include_alias=False)
```

Comment:
Can you have the include alias arg be passed in so this function is not assuming the caller already set aliases.